.drone.star

```diff
@@ -35,7 +35,7 @@ SONARSOURCE_SONAR_SCANNER_CLI = "sonarsource/sonar-scanner-cli:latest"
 THEGEEKLAB_DRONE_GITHUB_COMMENT = "thegeeklab/drone-github-comment:1"
 TOOLHIPPIE_CALENS = "toolhippie/calens:latest"
 
-OC10_VERSION = "10.9.1"
+OC10_VERSION = "10.10.0"
```

Comment:
@individual-it can we switch this back to the formerly used "latest" string (so we get a "free" upgrade and compatibility check and don't forget after the next OC10 release)?